extensions/kind/src/create-cluster.ts

```diff
@@ -57,7 +59,20 @@ export async function setupIngressController(clusterName: string) {
   );
 }
 
-export async function connectionAuditor(provider: string): Promise<AuditResult> {
+export async function connectionAuditor(provider: string, items: AuditRequestItems): Promise<AuditResult> {
+
+  const image = items["kind.cluster.creation.controlPlaneImage"];
+  if(image && !image.includes("@sha256:")) {
```

Comment:
```suggestion
  if(image && !image.includes('@sha256:')) {
```

this kind of issue is solved running `format:fix` and `lint:fix` rules (yarn)